src/main/java/org/jolokia/docker/maven/util/WaitUtil.java

```diff
@@ -176,6 +181,68 @@ public void cleanUp() { }
 
     // ====================================================================================================
 
+    /**
+     * Check whether a given TCP port is available
+     *
+     */
+    public static class TcpPortChecker implements WaitChecker {
+        private static final int TCP_TIMEOUT = 2000;
```

Comment:
Is 2 seconds not a bit long ? For the HTTP checker it 0.5 seconds. Remember, this time out is only for one try (there can be multiple tries until the overall timeout has reached).